service_configuration_lib/spark_config.py

```diff
@@ -292,6 +293,10 @@ def _adjust_spark_requested_resources(
             str(min(executor_instances, DEFAULT_K8S_BATCH_SIZE)),
         )
         user_spark_opts.setdefault('spark.kubernetes.executor.limit.cores', str(executor_cores))
+        user_spark_opts.setdefault(
+            'spark.scheduler.maxRegisteredResourcesWaitingTime',
+            str(10 + executor_instances * DEFAULT_RESOURCES_WAITING_TIME_PER_EXECUTOR // 60) + 'min',
```

Comment:
I think for safety maybe just do `ceil` rather than `\\`

Reply:
What do you mean by safety? `ceil` and `//` (or `floor`) doesn't make much difference in this scenario.